In-house patch to remove unnecessary dependencies from Nova's
requirements files. The specific reasons are as follows:

amqplib		No longer applicable
		(upstream commit 48591e83894354b57f0528ea98e6efe026acac28)

argparse	Not applicable to Solaris (nova-network specific)

paramiko	Not applicable to Solaris (PowerVM specific)

suds		Not applicable to Solaris (VMware specific)

--- nova-2013.2.3/nova.egg-info/requires.txt.orig	2014-04-03 11:55:51.000000000 -0700
+++ nova-2013.2.3/nova.egg-info/requires.txt	2014-05-24 23:15:07.242887922 -0700
@@ -1,6 +1,5 @@
 pbr>=0.5.21,<1.0
 SQLAlchemy>=0.7.8,<=0.7.99
-amqplib>=0.6.1
 anyjson>=0.3.3
 boto>=2.4.0,!=2.13.0
 eventlet>=0.13.0
@@ -14,8 +13,6 @@
 Paste
 sqlalchemy-migrate>=0.7.2
 netaddr
-suds>=0.4
-paramiko>=1.8.0
 pyasn1
 Babel>=1.3
 iso8601>=0.1.8

--- nova-2013.2.3/requirements.txt.orig	2014-04-03 11:49:46.000000000 -0700
+++ nova-2013.2.3/requirements.txt	2014-05-24 23:15:23.285062223 -0700
@@ -1,8 +1,6 @@
 pbr>=0.5.21,<1.0
 SQLAlchemy>=0.7.8,<=0.7.99
-amqplib>=0.6.1
 anyjson>=0.3.3
-argparse
 boto>=2.4.0,!=2.13.0
 eventlet>=0.13.0
 Jinja2
@@ -15,8 +13,6 @@
 Paste
 sqlalchemy-migrate>=0.7.2
 netaddr
-suds>=0.4
-paramiko>=1.8.0
 pyasn1
 Babel>=1.3
 iso8601>=0.1.8
